driver.go

```diff
@@ -281,18 +299,19 @@ func (l *lvmDriver) Mount(req *volume.MountRequest) (*volume.MountResponse, erro
 	}()
 
 	if l.count[req.Name] == 0 {
-		device := logicalDevice(vgName, req.Name)
+		device := logicalDevice(vol.VgName, req.Name)
 
 		if keyFile != "" {
-			if err := keyFileExists(keyFile); err != nil {
+			err := keyFileExists(keyFile)
+			if err != nil {
```

Comment:
just to define the `err` variable

Reply:
I don't follow, the `err` variable is already defined in the original snippet too.
Do you need it somewhere outside the following `if` clause.

Reply:
Arg, oversight on my end. But looking at the current semantics the `if err != nil {` inside defer would capture the err variable defined in `vgName, err := getVolumegroupName(l.vgConfig)` at the very beginning of Mount and not any following error since all are defined with :=